### PR TITLE
Update push.md

### PR DESCRIPTION
**Pull request by NimRegev**
Added new ECR fields role_arn, aws_session_name, and aws_duration_seconds to table

### CODE REVIEW

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/classic-ecr-cross-accoubt-push/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/classic-ecr-cross-accoubt-push/docs/

File: _docs/codefresh-yaml/steps/push.md [hunk 1]
```diff
@@ -56,6 +57,9 @@ step_name:
...
 | `region`                               | Relevant only for [Amazon ECR]({{site.baseurl}}/docs/integrations/docker-registries/amazon-ec2-container-registry/) integrations using either service accounts or explicit credentials. The names of the regions for which to perform cross-region replication. The names of the source region and the destination region name must be defined in separate steps.                                                                                                                                                                                | Optional                  |
+| `role_arn`                               | Relevant only for [Amazon ECR]({{site.baseurl}}/docs/integrations/docker-registries/amazon-ec2-container-registry/) integrations using either service accounts or explicit credentials. The role with the required permissions to use to pull the image. For example, `arn:aws:iam::<account-id>:role/<role-name>                                                                                                                                                                          | Required                  |
```
Comment (kostis-codefresh) [inline _docs/codefresh-yaml/steps/push.md:60]: It should be "optional". If the registry is not ECR then this field is not needed.

**Review by kostis-codefresh — APPROVED** [review]

File: _docs/codefresh-yaml/steps/push.md (hunk 1 above)
Comment (NimRegev) [reply _docs/codefresh-yaml/steps/push.md:60]: But we are assuming that it is ECR - we tell them that it's relevant only ECR registries and then it is required

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/classic-ecr-cross-accoubt-push/docs/

**Pull request merged**